### PR TITLE
TC_01.01.003 | Sign in to Jenkins > Sign in to Jenkins with valid credentials.

### PR DESCRIPTION
**Pull request by IliaKhudiakov**
https://trello.com/c/M8kzhCQ3/231-tc0101003-sign-in-to-jenkins-sign-in-to-jenkins-with-valid-credentials

### CODE REVIEW

**Review by verafes — APPROVED** [review]

File: cypress/e2e/SignInToJenkinsWithValidCredentials.cy.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress"/>
+
+const USERNAME = Cypress.env('local.admin.username');
+const PASSWORD = Cypress.env('local.admin.password');
+const LOCAL_PORT = Cypress.env('local.port');
+const LOCAL_HOST = Cypress.env('local.host');
+
+describe('Sign in to Jenkins', () => {
+
+    it('Sign in to Jenkins with valid credentials', function() {
```
Comment (S-Lanchik) [inline cypress/e2e/SignInToJenkinsWithValidCredentials.cy.js:10]: Add number to the test name

File: cypress/e2e/SignInToJenkinsWithValidCredentials.cy.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+       cy.get('a[href = "/logout"]').click()
+       cy.get('#j_username').type(USERNAME)
+       cy.get('#j_password').type(PASSWORD)
+       cy.get('button[type = "submit"]').click()
+       cy.url().should('eq', `http://${LOCAL_HOST}:${LOCAL_PORT}/`)
+       cy.get('h1').should('have.text', 'Welcome to Jenkins!')
```
Comment (S-Lanchik) [inline cypress/e2e/SignInToJenkinsWithValidCredentials.cy.js:16]: Put 'Welcome to Jenkins!' to fixtures

Comment (IliaKhudiakov) [reply cypress/e2e/SignInToJenkinsWithValidCredentials.cy.js:16]: I have corrected. Thanks!

Comment (S-Lanchik) [reply cypress/e2e/SignInToJenkinsWithValidCredentials.cy.js:16]: I don't see your corrections. Also, use recommendations from your other PR.

**Review by S-Lanchik — APPROVED** [review]

**Pull request merged**